<a href="https://colab.research.google.com/github/aparna-2001/machine_learning_using_python/blob/main/Binomial_distribution.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
from scipy import stats

Fashion Transform Online (FTO) is an e-commerce company that sells women apparel. It is observed that 10% of the customers return the items purchased by them for many reasons( such as size, color, and material mismatch). On a specific day, 20 customers purchase the items from FTO. Calculate:
*  probability that exactly 5 customers will return the items,
 *  probability that maximum of 5 customers will return the items,
 * probability that more than 5 customers will return the items purchased by them,
 * average number of customers who are likely to return the items, and the variance and the standard deviation of the number of returns.

In [3]:
#1

stats.binom.pmf(5,20, 0.1)

np.float64(0.031921361119954424)



*   probability that exactly 5 customers will return is 3%



In [4]:
#2

stats.binom.cdf(5, 20, 0.1)

np.float64(0.988746865835491)

* probability that a maximum of 5 customers will return is 0.9887

In [5]:
#3

1 - stats.binom.cdf(5, 20, 0.1)

np.float64(0.011253134164509015)

* Probability that more than 5 customers will return the item is 1%

In [8]:
#4

avg, std = stats.binom.stats(20, 0.1)
print('avg: ', avg)
print('std: ', std)

avg:  2.0
std:  1.8
